Libplanet.Extensions.Cocona/Commands/KeyCommand.cs

```diff
@@ -66,23 +86,46 @@ public KeyCommand()
             }
         }
 
-        [Command(Description = "Import a raw private key.")]
+        [Command(Description = "Import a raw private key or Web3 Secret Storage.")]
         public void Import(
             [Argument(
                 "PRIVATE-KEY",
-                Description = "A raw private key to import in hexadecimal string."
+                Description = "A raw private key in hexadecimal string, or path to Web3 Secret " +
+                              "Storage to import"
             )]
             string key,
             PassphraseParameters passphrase,
             [Option(
-                Description = "Print the created private key as Web3 Secret Storage format."
+                Description = "Import Web3 Secret Storage key."
             )]
             bool json = false,
             [Option(Description = "Do not add to the key store, but only show the created key.")]
-            bool dryRun = false
+            bool dryRun = false,
+            [Option(Description = "Path to key store")]
+            string? path = null
         )
         {
-            Add(key, passphrase, json, dryRun, key, false);
+            ChangeKeyStorePath(path);
+            if (json)
+            {
+                try
+                {
+                    ProtectedPrivateKey ppk = ProtectedPrivateKey.FromJson(ValidateJsonPath(key));
+                    PrintKeys(new[] { (Add(ppk, dryRun), ppk) });
+                }
+                catch (Exception)
+                {
+                    Utils.Error("Couldn't load ppk from json.");
+                }
```

Comment:
It should pass if the `CommandExitedException` was thrown. And it should throw `CommandExitedException`  for the exit code. If it doesn't throw, it'll return `0` like successful execution.

```suggestion
                catch (CommandExitedException)
                {
                    throw;
                }
                catch (Exception)
                {
                    throw Utils.Error("Couldn't load ppk from json.");
                }
```